### PR TITLE
Install and setup Linters in 'Hello Microverse!' project.

### PR DESCRIPTION
**Pull request by Javivy**
Add linters
Create HTML and CSS file
Create .gitignore

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    name: Webhint
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Webhint
+        run: |
+          npm install --save-dev hint@7.x
```
Comment (Kamba56) [inline .github/workflows/linters.yml:31]: - [ ] Kindly setup webhint correctly. Please copy the files from [this](https://github.com/microverseinc/linters-config/tree/master/html-css) repo into your repository and run `npm install --dave-dev hint@7.x` in your terminal. Please do not delete the `package.lock.json`, `package.json` and `.hintrc` files.

File: .github/workflows/linters.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    name: Stylelint
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
```
Comment (Kamba56) [inline .github/workflows/linters.yml:45]: - [ ] Kindly set up stylelint correctly. Please copy the files from [this](https://github.com/microverseinc/linters-config/tree/master/html-css) repo into your repository and run ` npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x` in your terminal. Please do not delete the `package.lock.json`, `package.json`, and `.stylelintrc.json` files.

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @Javivy, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
SoCloseKeeganMichaelKeyGIF

## Highlights

- [x] Meaningful commit messages
- [x] All linters checks passed
- [x] Descriptive summary of pull request

## Required Changes ♻️
- [ ] Please edit your `README.md` file to be more descriptive. This will make your repo more professional.
- [ ] Kindly edit your pull request title. The pull request title should be descriptive, describing the implemented change. You can edit the pull request title by clicking the `edit` button next to it.

_Check the comments under the review for more changes._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [Optional] Kindly change the name of your branch to a [feature or action](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md#feature-branches) to make your work look more professional. You can rename your branch by following [this](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/renaming-a-branch) steps.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @Javivy, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
 
## [Optional] suggestions:
>- [ ] Next time, and for future Pull Requests, please kindly add a more descriptive PR title.
> See a demo below:
pull_request_desc

 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**